src/test/java/org/broadinstitute/hellbender/tools/examples/ExampleVariantWalkerIntegrationTest.java

```diff
@@ -0,0 +1,32 @@
+package org.broadinstitute.hellbender.tools.examples;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.tools.IntegrationTestSpec;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+public final class ExampleVariantWalkerIntegrationTest extends CommandLineProgramTest {
+    @Override
+    public String getTestedClassName() {
+        return ExampleVariantWalker.class.getSimpleName();
+    }
```

Comment:
hmm, not sure. I like to be able to quickly navigate from the test to the tested class, which this reference enables. Without this reference I need to open a special dialog to type in the class name etc. Plus, the class then looks unused to the IDE.